virtcontainers/pkg/types/types.go

```diff
@@ -5,6 +5,18 @@
 
```

Comment:
Nit: typo in the commit message `descrptions`. Don't forget to run type checker over commit message. :)

Reply:
Fixed, thanks.